cmd/users.go

```diff
@@ -51,7 +58,12 @@ func userListCmd() *cobra.Command {
 	}
 
 	flags := cmd.Flags()
-	flags.StringArrayVarP(&options.keywords, "keywords", "", []string{}, "search keywords")
+	flags.StringVar(&options.keyword, "keywords", "", "comma or space-separated string to search in secret's names")
+	flags.IntVar(&options.offset, "offset", 0, "where to start fetching the items")
+	flags.IntVar(&options.limit, "limit", 50, "number of items to return")
+	flags.StringVar(&options.sortdir, "sortdir", "", "sort direction, ASC or DESC (default ASC)")
+	flags.StringVar(&options.sortkey, "sortkey", "", "sort object by property: source, email, principal, full_name.")
+	flags.StringArrayVar(&options.UserIDs, "userid", []string{}, "list of users IDs.")
```

Comment:
Why would i add "ASC" while it's by default like that?

Reply:
It is best practice to explicitly provide all params (if possible) to avoid side effects. The side effects come from the implicit dependency on how the backend would consider "default". The backend behaviour could change without prior notice (however unlikely that might be). It is a best practice recommendation.

Reply:
OK, i see. Thanks!